libdeno/libdeno_test.cc

```diff
@@ -294,7 +294,7 @@ TEST(LibDenoTest, ModuleResolutionFail) {
     // Do not call deno_resolve_ok();
   };
   Deno* d = deno_new(deno_config{0, empty, empty, nullptr, resolve_cb});
-  EXPECT_FALSE(deno_execute_mod(d, d, "a.js", mod_a));
+  EXPECT_FALSE(deno_execute_mod(d, d, "a.js", mod_a, 0));
```

Comment:
It would be nice to have a libdeno test exercising the `is_prefetch` option (or whatever you end up naming it)

Suggest something like this (please change naming)
```C++
TEST(LibDenoTest, ModuleResolutionPrefetch) {
  static int count = 0;
  auto resolve_cb = [](void* user_data, const char* specifier,
                       const char* referrer) {
    EXPECT_STREQ(specifier, "b.js");
    EXPECT_STREQ(referrer, "a.js");
    count++;
    auto d = reinterpret_cast<Deno*>(user_data);
    deno_resolve_ok(d, "b.js", mod_b);
  };
  Deno* d = deno_new(deno_config{0, empty, empty, nullptr, resolve_cb});
  EXPECT_TRUE(deno_execute_mod(d, d, "a.js",
    "import { retb } from 'b.js'\n"
    "throw Error('unreachable');" , 1));
  EXPECT_EQ(count, 1);
  deno_delete(d);
}
```